libraries/botbuilder-dialogs-adaptive/src/actions/deleteProperties.ts

```diff
@@ -37,7 +37,7 @@ export class DeleteProperties<O extends object = {}> extends Dialog<O> {
 
     /**
      * Called when the dialog is started and pushed onto the dialog stack.
```

Comment:
Same changes for the other two Actions PRs.
We should add the dialog reference in multiple files

Reply:
Solved!